tests/unit/src/test/resources/foldingRange-scala3-foldLineOnly/expect/Main.java

```diff
@@ -41,13 +42,13 @@ String hello(String str) >>region>>{
             return "asssd";
         }<<region<<
 
-        void openingCommentInStr() >>region>>{
```

Comment:
Why did we lose a region here? That should be 3 lines with all the {}

Same below

Reply:
I counted only the lines that actually get folded, so:
```scala
def a = {
  1
}
```
is one
```scala
def a =
  1
```
is also one. Think it's a bit more intuitive than if these had two and one respectively. I can offset by one, so both count as 2, would that be better?

Reply:
ok, let's try this solution. I don't really have preference and it looks sensible. Thanks for explaining!